detox/src/artifacts/video/SimulatorRecordVideoPlugin.js

```diff
@@ -11,6 +11,7 @@ class SimulatorRecordVideoPlugin extends VideoArtifactPlugin {
     super(config);
 
     this.appleSimUtils = config.appleSimUtils;
+    this.codec = config.codec;
```

Comment:
The config does not currently have the codec passed to it. Advise on the best way to do this would be helpful.

Reply:
@noomorph @rotemmiz Pls advice.